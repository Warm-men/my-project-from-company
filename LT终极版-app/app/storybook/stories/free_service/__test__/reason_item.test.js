import React from 'react'
import render from 'react-test-renderer'
import ReasonItem from '../reason_item'
describe('open freeService successful page', () => {
  it('renders correctly', () => {
    expect(render.create(<ReasonItem />).toJSON()).toMatchSnapshot()
  })
})
