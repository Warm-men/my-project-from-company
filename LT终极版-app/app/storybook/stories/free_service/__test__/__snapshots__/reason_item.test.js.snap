// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`open freeService successful page renders correctly 1`] = `
<View
  accessible={true}
  isTVSelectable={true}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#FDEDE9",
      "borderColor": "#E85C40",
      "borderRadius": 91,
      "borderWidth": 1,
      "height": 44,
      "justifyContent": "center",
      "marginTop": 16,
      "opacity": 1,
      "width": 630,
    }
  }
>
  <Text
    style={
      Object {
        "color": "#E85C40",
        "fontSize": 14,
      }
    }
  />
</View>
`;
